docs/notebooks/image_to_image_arcface.ipynb

```diff
@@ -0,0 +1,362 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "p8jc8EyfruKw"
+      },
+      "source": [
+        "# Image-to-Image Search with ArcFaceLoss\n",
+        "\n",
+        "<a href=\"https://colab.research.google.com/drive/1ZS9FmnR9FzO_JYGCPazFM7TcMNQl51xM?usp=sharing\"><img alt=\"Open In Colab\" src=\"https://colab.research.google.com/assets/colab-badge.svg\"></a>\n",
+        "\n",
+        "Searching visually similar images with image queries is a very popular use case. However, using pre-trained models does not deliver the best results – the models are trained on general data that lack the particularities of your specific task. Here's where Finetuner comes in! It enables you to accomplish this easily.\n",
+        "\n",
+        "This guide will demonstrate how to fine-tune a ResNet model using the `ArcFaceLoss` function.\n",
+        "\n",
+        "*Note, please consider switching to GPU/TPU Runtime for faster inference.*\n",
+        "\n",
+        "## Install"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "VdKH0S0FrwS3"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install 'finetuner[full]'"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7EliQdGCsdL0"
+      },
+      "source": [
+        "## Task\n",
+        "\n",
+        "More specifically, we will fine-tune ResNet50 on [Stanford Cars Dataset](http://ai.stanford.edu/~jkrause/cars/car_dataset.html).\n",
+        "This dataset consists of 196 classes across 16184 docuemnts in total.\n",
+        "\n",
+        "Each class of the dataset consists of roughly 82 documents, in order to group the documents of each class together and move separate documents of different classes apart, we use the `ArcFaceLoss` function. For more information on how this loss function works, see [Advanced Losses and Optimizers](https://finetuner.jina.ai/advanced-topics/advanced-losses-and-optimizers/)\n",
+        "\n",
+        "After fine-tuning, documents from each class should have similar embeddings, distinct from documents of other classes."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "M1sii3xdtD2y"
+      },
+      "source": [
+        "## Data\n",
+        "\n",
+        "Our journey starts locally. We have to prepare the data and push it to the Jina AI Cloud and Finetuner will be able to get the dataset by its name. For this example,\n",
+        "we already prepared the data, and we'll provide the names of training data (`stanford-cars-train`) directly to Finetuner.\n",
+        "\n",
+        "```{important} \n",
+        "We don't require you to push data to the Jina AI Cloud by yourself. Instead of a name, you can provide a `DocumentArray` and Finetuner will do the job for you.\n",
```

Comment:
```suggestion
        "You don't have to push your data to the Jina AI Cloud before fine-tuning. Instead of a name, you can provide a `DocumentArray` and Finetuner will do upload your data directly.\n",
```